book_source/05_developer_guide/03_git/01_using-git.Rmd

```diff
@@ -1,25 +1,7 @@
 ## Using Git
 
-
-This document describes the steps required to download PEcAn, make changes to code, and submit your changes.
```

Comment:
There's a whole section of text here that was at the top of the page that provided a roadmap to the rest of the information below. Other than removing outdated references to Redmine it's not clear to me why this text was removed.